build.gradle

```diff
@@ -22,6 +22,12 @@ subprojects {
     tasks.withType(JavaCompile) { options.encoding = 'UTF-8' }
     group = 'com.google.gct'
 
+    task jarSources(type:Jar) {
```

Comment:
I can't really find any other way to do this. But by default no sources jar is generated.